plugin/minimal/setup.go

```diff
@@ -0,0 +1,27 @@
+package minimal
+
+import (
+	"github.com/coredns/caddy"
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+)
+
+func init() {
+	caddy.RegisterPlugin("minimal-responses", caddy.Plugin{
```

Comment:
there is a 1-line short cut for this now

Reply:
I see, is this the function `plugin.Register("minimal", setup)`

Reply:
Made the change